modules/match.js

```diff
@@ -17,26 +18,30 @@ const createHistory = useRoutes(useBasename(createMemoryHistory))
  */
 function match({
   routes,
+  path,
   location,
-  parseQueryString,
-  stringifyQuery,
-  basename
+  ...options
 }, callback) {
   invariant(
```

Comment:
Do we really need this `invariant`? It's just going to not work without a location/path, no?